colander/__init__.py

```diff
@@ -1848,7 +1850,9 @@ class _SchemaNode(object):
       is only meaningful to higher-level systems such as Deform.
 
     Arbitrary keyword arguments remaining will be attached to the node
-    object unmolested.
+    object unmolested.  (NB: be **careful** with this feature as the
+    process used will overwrite existing attributes *as well as
+    methods*.)
```

Comment:
I don't see a point in this message. It's a bug that this happens and instead of documenting it we should prevent it from overwriting unexpected attributes.